releases/_posts/2023-04-13-spark-release-3-4-0.md

```diff
@@ -0,0 +1,420 @@
+* Add Dataset.as(StructType) ([SPARK-39625](https://issues.apache.org/jira/browse/SPARK-39625))
+* Support parameterized SQL ([SPARK-41271](https://issues.apache.org/jira/browse/SPARK-41271), [SPARK-42702](https://issues.apache.org/jira/browse/SPARK-42702))
+* Add unpivot / melt ([SPARK-38864](https://issues.apache.org/jira/browse/SPARK-38864), [SPARK-39876](https://issues.apache.org/jira/browse/SPARK-39876))
+* Support "lateral column alias references" ([SPARK-27561](https://issues.apache.org/jira/browse/SPARK-27561))
+* Support result offset clause ([SPARK-28330](https://issues.apache.org/jira/browse/SPARK-28330), [SPARK-39159](https://issues.apache.org/jira/browse/SPARK-39159))
+* Support Timestamp without time zone data type ([SPARK-35662](https://issues.apache.org/jira/browse/SPARK-35662))
+* Support scalar subquery in time travel ([SPARK-39306](https://issues.apache.org/jira/browse/SPARK-39306))
+* Make Catalog API be compatible with 3-layer-namespace ([SPARK-39235](https://issues.apache.org/jira/browse/SPARK-39235))
+* Support timestamp in seconds for TimeTravel using Dataframe options ([SPARK-39633](https://issues.apache.org/jira/browse/SPARK-39633))
+* Add SparkSession.config(Map) ([SPARK-40163](https://issues.apache.org/jira/browse/SPARK-40163))
+* Support changing session catalog's default database ([SPARK-35242](https://issues.apache.org/jira/browse/SPARK-35242))
+* Protobuf support for Spark - from_protobuf AND to_protobuf ([SPARK-40654](https://issues.apache.org/jira/browse/SPARK-40654))
+* Add WHEN NOT MATCHED BY SOURCE clause to MERGE INTO ([SPARK-40921](https://issues.apache.org/jira/browse/SPARK-40921))
+* Relax ordering constraint for CREATE TABLE column options ([SPARK-40944](https://issues.apache.org/jira/browse/SPARK-40944))
+* SQL Equivalent for Dataframe overwrite command ([SPARK-40956](https://issues.apache.org/jira/browse/SPARK-40956))
+* Support Generate with no required child output to host outer references ([SPARK-41441](https://issues.apache.org/jira/browse/SPARK-41441))
+* ORDER BY ALL ([SPARK-41637](https://issues.apache.org/jira/browse/SPARK-41637))
+* GROUP BY ALL ([SPARK-41635](https://issues.apache.org/jira/browse/SPARK-41635))
+* Add flatMapSortedGroups and cogroupSorted ([SPARK-38591](https://issues.apache.org/jira/browse/SPARK-38591))
+* Support subqueries with correlated non-equality predicates ([SPARK-36114](https://issues.apache.org/jira/browse/SPARK-36114))
+* Support subqueries with correlation through UNION/INTERSECT/EXCEPT ([SPARK-36124](https://issues.apache.org/jira/browse/SPARK-36124))
+
+#### ANSI Compliance
+
+* ANSI SQL mode: always return null on invalid access to map column ([SPARK-40066](https://issues.apache.org/jira/browse/SPARK-40066))
+* Support double quoted identifiers ([SPARK-40585](https://issues.apache.org/jira/browse/SPARK-40585))
+* ANSI SQL mode: Round/Bround should return an error on integer overflow [SPARK-42045](https://issues.apache.org/jira/browse/SPARK-42045))
```

Comment:
nit: The left paren is missing before `[SPARK-`.

Reply:
Updated.